resources/sts/4.14/hypershift/sts_extended_hcp_support_permission_policy.json

```diff
@@ -1,115 +1,407 @@
+        "ec2:DescribeClientVpnAuthorizationRules",
+        "ec2:DescribeClientVpnConnections",
+        "ec2:DescribeClientVpnRoutes",
+        "ec2:DescribeClientVpnTargetNetworks"
+      ],
+      "Resource": "arn:aws:ec2:*:*:client-vpn-endpoint/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeClientVpnEndpoint",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeClientVpnEndpoints"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeVpnConnectionsAndGateways",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeVpnConnections",
+        "ec2:DescribeVpnGateways"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeAndModifyInstanceAttribute",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeInstanceAttribute",
+        "ec2:ModifyInstanceAttribute"
+      ],
+      "Resource": "arn:aws:ec2:*:*:instance/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
+        }
+      }
+    },
+    {
+      "Sid": "DescribeLocalGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeLocalGatewayRouteTables",
+        "ec2:DescribeLocalGatewayRouteTableVirtualInterfaceGroupAssociations",
+        "ec2:DescribeLocalGatewayRouteTableVpcAssociations",
+        "ec2:DescribeLocalGateways",
+        "ec2:DescribeLocalGatewayVirtualInterfaceGroups",
+        "ec2:DescribeLocalGatewayVirtualInterfaces"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "DescribeTransitGateway",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:DescribeTransitGatewayAttachments",
+        "ec2:DescribeTransitGatewayConnectPeers",
+        "ec2:DescribeTransitGatewayConnects",
+        "ec2:DescribeTransitGatewayMulticastDomains",
+        "ec2:DescribeTransitGatewayPeeringAttachments",
+        "ec2:DescribeTransitGatewayRouteTables",
+        "ec2:DescribeTransitGateways",
+        "ec2:DescribeTransitGatewayVpcAttachments",
+        "ec2:GetTransitGatewayAttachmentPropagations",
+        "ec2:GetTransitGatewayPrefixListReferences",
+        "ec2:GetTransitGatewayRouteTableAssociations",
+        "ec2:GetTransitGatewayRouteTablePropagations"
+      ],
+      "Resource": "*"
+    },
+    {
+      "Sid": "ManageMulticastDomain",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:GetTransitGatewayMulticastDomainAssociations",
+        "ec2:SearchTransitGatewayMulticastGroups"
+      ],
+      "Resource": "arn:aws:ec2:*:*:transit-gateway-multicast-domain/*"
+    },
+    {
+      "Sid": "SearchTransitGatewayRoutes",
+      "Effect": "Allow",
+      "Action": [
+        "ec2:SearchTransitGatewayRoutes"
+      ],
+      "Resource": "arn:aws:ec2:*:*:transit-gateway-route-table/*",
+      "Condition": {
+        "StringEquals": {
+          "aws:ResourceTag/red-hat-managed": "true"
```

Comment:
Do these all have this tag?

Reply:
the customer transitgateway won't have this tag. removing it.